src/main/scala/net/snowflake/spark/snowflake/io/CloudStorageOperations.scala

```diff
@@ -723,6 +723,14 @@ sealed trait CloudStorage {
       "Negative test to raise error when uploading data to GCS"
     )
 
+    // When attempt number is smaller than 2, throw exception
```

Comment:
It looks it is no necessary to keep two flags for this case. Can TH_GCS_UPLOAD_RAISE_EXCEPTION and TH_UPLOAD_RAISE_EXCEPTION_WITH_COUNT merged as one?

Reply:
Merged two flags into one